Update README.md

colinfrederickson opened the pull request:
Includes two items we discussed on 11/30 for inclusion into the README

colinfrederickson committed:
Update README.md

paddymul commented on README.md:
````diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
 ```
+
+
+**CF Notes** - based on 11/30 convo do we want to include specifics on:
````
I would add these to the FAQ section

paddymul commented on README.md:
```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
+
+**CF Notes** - based on 11/30 convo do we want to include specifics on:
+
+-  Are there character limits for error messages?
```
I don't know.  Want to try deploying a sheet with a reallllly long message and see where it fails.  test around 256 (smaller larger), 512, and 1024.  The backend might save the message and the frontend might do something wonky on display.

colinfrederickson replied:
this is more of a presentation issue on the front end, removing from FAQ.

paddymul commented on README.md:
```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
+**CF Notes** - based on 11/30 convo do we want to include specifics on:
+
+-  Are there character limits for error messages?
+-  Addressing messaging around fields set via data hook (ex: we have a data hook that populates an empty field. Upon landing in review, that field is populated with a value and a message.  If another field on that record is populated, when data hooks are run again the field remains populated but without an error message letting the user know the field had been initially transformed.
```
Add this to the FAQ for now.

This hints at the DataHook lifecycle, which is implied right now, but should be made more clear.

colinfrederickson committed:
Begin to Address Paddy's Comments

Addresses 11/30 comments from Paddy. Still needs results from error message testing.

colinfrederickson committed:
Removed Character Limits

Removed Character Limit piece from FAQ
